frontend/css/theming.css

```diff
@@ -1,5 +1,7 @@
 :root {
     --blurred-background-color: rgba(255, 255, 255, 0.2);
+    --white-background-color: white;
```

Comment:
Just a general note on naming, personally I prefer functional/descriptive names of this kind of variables. If `white` was a reference to a custom colour itself, I would be fine with naming that `white`. 

The `--white-background-color` though, I would name something like `--main-background-color`, since it describes how it's supposed to be used. If you then were to change the actual colour to something else (greyish perhaps), the name would still make sense since it doesn't describe the colour shade itself. 

Not everyone agrees with this approach though but that's how I like to do it. :)

Reply:
Thanks Sara, that makes sense. I've changed --white-background-color to --main-background-color